File: src/TheFipster.Munchkin.Gaming.Domain/Models/Hero.cs
using System.Collections.Generic;

namespace TheFipster.Munchkin.Gaming.Domain
{
    public class Hero
    {
        public Hero() { }

        public Hero(Player player)
        {
            Player = player;
            Level = 1;
            Races = new List<string>();
            Classes = new List<string>();
        }

        public Player Player { get; set; }
        public int Level { get; set; }
        public int Bonus { get; set; }
        public ICollection<string> Races { get; set; }
        public ICollection<string> Classes { get; set; }
    }
}

File: src/TheFipster.Munchkin.Spa/src/app/interfaces/hero.ts
import { Player } from './player';

export class Hero {
    player: Player;
    level: number;
    bonus: number;
    races: string[];
    classes: string[];

    constructor(player: Player) {
        this.player = player;
        this.level = 1;
        this.bonus = 0;
        this.races = [];
        this.classes = [];
    }
}
